src/ProductBundle/Controller/CollectionController.php

```diff
@@ -109,4 +173,40 @@ public function listSideMenuCollectionsAction($collection = null, $depth = 1, $d
           'deep' => $deep + 1,
         ]);
     }
+
+    protected function getCurrencyDetector(): CurrencyDetectorInterface
```

Comment:
Either make this final or private

Reply:
i don't understand why only this method but it's done